cypress/integration/App.spec.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
```

Comment:
We should declare globals and not disable rules.

Reply:
It's giving me error on 
```
'describe' is not defined.eslint(no-undef)
```

And also on `beforeEach`, `cy`, and `it`. I couldn't find a fix for it.
Technically those are undefined in that file but tests work. How to deal with that?

Reply:
[Use this plugin](https://github.com/cypress-io/eslint-plugin-cypress). Let me know if you need help.

Reply:
That worked perfectly! I can't believe I didn't find that plugin. I 'll keep this in mind in-general for what to look for when I see such problem. Thanks! 😃